backend/src/LibExecution/Dval.fs

```diff
@@ -9,13 +9,13 @@ module VT = ValueType
 
 let int (i : int) = DInt(int64 i)
 
-let int8 (i : int) = DInt8(int8 i)
+let int8 (i : int8) = DInt8(int8 i)
```

Comment:
misc, we shouldn't need to call the `int8` fn here, and in similar fns just below

Reply:
Fixed, thanks

Reply:
Fixed, thanks